tasks-schema.json

```diff
@@ -356,24 +466,175 @@
                         },
                         "value": {
                           "type": "object",
+                          "description": "Object defining the characteristics of valid values of the cumulative distribution function at a given target value.",
                           "properties": {
                             "type": {
+                              "description": "Data type of cumulative distribution function values.",
+                              "example": "double",
                               "enum": [
                                 "numeric", "integer", "double"
                               ]
                             },
                             "minimum": {
+                              "description": "The minimum inclusive valid cumulative distribution function value",
```

Comment:
```suggestion
                              "description": "The minimum inclusive valid cumulative distribution function value.",
```

Reply:
Indeed. I wasn't sure whether such a use case would exist so left it as a possibility. But happy to hard code 0 & 1 into the checks